### ISSUE TITLE
Locking of sources breaks keys when installing Pods multiple times

### ISSUE THREAD

**Issue by neonichu** [orta/cocoapods-keys#87]
```bash
Errno::EACCES - Permission denied - /Users/boris/Sources/GIFs-1/objc/Pods/CocoaPodsKeys/GIFsKeys.h
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-keys-1.3.1/lib/plugin.rb:32:in `write'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-keys-1.3.1/lib/plugin.rb:32:in `setup'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-keys-1.3.1/lib/plugin.rb:86:in `install!'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command/project.rb:101:in `run'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/claide-0.8.1/lib/claide/command.rb:312:in `run'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-0.37.2/lib/cocoapods/command.rb:46:in `run'
/Users/boris/Software/ruby-200/lib/ruby/gems/2.0.0/gems/cocoapods-0.37.2/bin/pod:44:in `<top (required)>'
/Users/boris/Software/ruby-200/bin/pod:23:in `load'
/Users/boris/Software/ruby-200/bin/pod:23:in `<main>'
```

I guess cp-keys has to unlock its sources before rewriting them.

**Comment by segiddins** [comment]
Is this fixed using CocoaPods master?

**Comment by orta** [comment]
this isn't something I've seen FWIW, and we use Keys everywhere.

**Comment by neonichu** [comment]
Seems like it is indeed fixed in master, but appears in 0.37.2
